### PR TITLE
Full Spanish Translation

### PR DESCRIPTION
**Pull request by troyodk**
Full Spanish Translation.

### CODE REVIEW

**Comment by pstorch** [comment]
@troyodk thanks for your pull request. The CI reports that there is a `</string>` missing somewhere.
Can you have a look?

```
> /home/travis/build/RailwayStations/RSAndroidApp/app/src/main/res/values-es/strings.xml:208:3: Error: The element type "string" must be terminated by the matching end-tag "</string>".
```

**Comment by rakvium** [comment]
@troyodk you have extra or unmatched `>` symbol somewhere in the file.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,5 +1,208 @@
...
+  <string name="license_info_not_readable">Información de la licencia ilegible por el momento</string>
+  <string name="station_not_found">Estación no encontrada</string>
+  <string name="last_update_at">Ultima vez actualizado en: </string>
+  <string name="station_update_failed">Actualización de estacion fallida:</string>
+  <string name="stationId">StationNbr: </string>
+  <string name="need_to_allow_location_access">Necesita acceder a dar permisos locales</string>
+  <string name="my_position">Mi posición actual es : </string>
+  <string name="chat_notification_enabled">Ha activado satisfactoriamente las notificaciones del chat</string>
+  <string name="chat_notification_disabled">Ha desactivado satisfactoriamente las notificaciones del chat</string>
+  <string name="unable_to_create_folder_structure">Fallo al crear estructura de direcciones/string>
```
Comment (rakvium) [inline app/src/main/res/values-es/strings.xml]: @troyodk, I have found it! You have missed `<` before `/string` here. If you fix it, all will be good.

**Comment by troyodk** [comment]
Hello! Sorry it was 3 a.m back here in México when I created the PR, I will fix it and do a quick grammar check

**Comment by pstorch** [comment]
Muchas gracias :+1:

**Comment by troyodk** [comment]
No hay problema!

**Pull request merged**